### ISSUE TITLE
Mega Menu: Accessibility - Issue related to Search edit combo list box

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#3585]
## Detailed description

On iPhone Voice over 

URL - https://ibmdotcom-react-canary.mybluemix.net/

Issue :
After entering the search term visually the options are appearing but screen reader users are not able to navigate through the options within the combo box

Expected :- the user should be able to navigate through the options using left and right swipe and make the selection.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/de62036a-64aa-4d92-960a-121728e06419)

**Comment by RaphaelAmadeu** [comment]
@Praveen-sr 

I've tested it in my phone using talk back, and I was able to navigate through the list by swiping slowly up and down, after getting to the list-box.

This behavior is set by an external library that we use for the suggestions, that is accessibility compliant, as we can check here (https://github.com/moroshko/react-autosuggest).

Do we need to override their up and down selection for side swapping?
CC: @jeffchew @annawen1

**Comment by RobertaJHahn** [comment]
@jeffchew Changed the severity to SEv 3 based on Raph's response.  Do we still need in v1.10.0?

**Comment by RobertaJHahn** [comment]
Moving to icebox because a technical solution is not available at this time.  Search would have to be completely reworked / designed in order to find a solution.

**Comment by RobertaJHahn** [comment]
Will be moving away from support of Pure React components.  Closing this epic and all linked issues.
